### PR TITLE
Fix for uncaught attemptChangeSize Error

### PR DESCRIPTION
**Pull request by zhouyx**

### CODE REVIEW

**Comment by zhouyx** [comment]
I test ads example locally, don't see uncaught errors now
PTAL

File: extensions/amp-list/0.1/amp-list.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ export class AmpList extends AMP.BaseElement {
       const scrollHeight = this.container_./*OK*/scrollHeight;
       const height = this.element./*OK*/offsetHeight;
       if (scrollHeight > height) {
-        this.attemptChangeHeight(scrollHeight);
+        this.attemptChangeHeight(scrollHeight).then(() => {}, () => {});
```
Comment (jridgewell) [inline extensions/amp-list/0.1/amp-list.js]: This just needs a `#catch`.

Comment (zhouyx) [inline extensions/amp-list/0.1/amp-list.js]: fixed

**Comment by jridgewell** [comment]
LGTM.

File: extensions/amp-ad/0.1/amp-ad-api-handler.js [hunk 2]
```diff
@@ -156,7 +156,7 @@ export class AmpAdApiHandler {
   updateSize_(height, width) {
     const targetOrigin =
           this.iframe_.src ? parseUrl(this.iframe_.src).origin : '*';
-    this.baseInstance_.attemptChangeSize(height, width, () => {
+    this.baseInstance_.attemptChangeSize(height, width).then(() => {
```
Comment (lannka) [inline extensions/amp-ad/0.1/amp-ad-api-handler.js]: sorry to ask: didn't we have a test to cover this ?

Comment (zhouyx) [inline extensions/amp-ad/0.1/amp-ad-api-handler.js]: We have tests for `updateSize_` but we only test that `attemptChangeSize(height, width)` are been called. 
You are right  we need to test `#postMessage`  is called. 
#4256

**Pull request merged**